gradle/verification-metadata.xml

```diff
@@ -3022,29 +3022,29 @@
             <sha256 value="a82ac5bc24bcbf6ba9eb70f334d6782e25245c8da36d9848ad553b5b7b68efd1" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.bouncycastle" name="bcpkix-jdk15on" version="1.64">
-         <artifact name="bcpkix-jdk15on-1.64.jar">
-            <sha256 value="84669138b1d99143e2c009024f67824ab8d3edb9b05b7591f5ebfb020a4bda71" origin="Generated by Gradle"/>
+      <component group="org.bouncycastle" name="bcpkix-jdk18on" version="1.76">
+         <artifact name="bcpkix-jdk18on-1.76.jar">
+            <sha256 value="935a388854c329f9a6f32708f30c90045d2f91294fa687281273145d4cf9834a" origin="Generated by Gradle"/>
          </artifact>
       </component>
       <component group="org.bouncycastle" name="bcprov-jdk15on" version="1.60">
          <artifact name="bcprov-jdk15on-1.60.jar">
             <sha256 value="7f1a0e6badab38666f8467a9a0ee96656b2f8ec8623867ed34f3cdc173b7ee07" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.bouncycastle" name="bcprov-jdk15on" version="1.64">
```

Comment:
I was able to remove many of the elder bouncy castle jars from here, however, the hdfs test frame work and (I believe) gradle plugins used for the build still explicitly use elder versions of bouncy castle. The versions we ship and/or test with FIPS have been upgraded and can be confirmed with the gradlew scan:: https://scans.gradle.com/s/xask2dbwyb7hk/dependencies?dependencies=bouncycastle&expandAll